chain/jsonrpc/test-utils/src/lib.rs

```diff
@@ -39,12 +44,19 @@ pub fn start_all_with_validity_period_and_no_epoch_sync(
     );
 
     let addr = format!("127.0.0.1:{}", open_port());
+    #[cfg(feature = "adversarial")]
+    let pm1 = {
+        let (pm2, _, _) = make_peer_manager("test2", open_port(), vec![("test1", open_port())], 10);
 
+        pm2.start()
```

Comment:
Why `pm1` and `pm2`? I feel those are two quite different objects.

Reply:
renamed